api/mobile/api.go

```diff
@@ -12,6 +12,7 @@ import (
 
 // UserInformationRepresentation struct
 type UserInformationRepresentation struct {
+	RealmName      *string                        `json:"realmName,omitempty"`
```

Comment:
Should we call it DisplayName to avoid confusion later ?

Reply:
Renamed: RealmDisplayName